From f80155c18e31020ad166e0f414e2b1c330c98992 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Wed, 06 Nov 2019 13:39:49 +0200
Subject: [PATCH] [SW-0] hl: define bases only for ARM64

These macros are specific to our ARM64 design

Note: macro __aarch64__ defined as part of toolchain

Change-Id: I073a6282a2386341b4e30c97bfaf8d2ca558ced9
---

diff --git a/common/preboot.h b/common/preboot.h
index 36b5041..3278135 100644
--- a/common/preboot.h
+++ b/common/preboot.h
@@ -11,10 +11,13 @@
 #define BIT(nr)					(1 << (nr))
 #define ARRAY_SIZE(x)				(sizeof(x) / sizeof((x)[0]))
 
+#ifdef __aarch64__
 #define HL_SPI_BASE				0x7FF8000000ull
 #define HL_CFG_BASE				0x7FFC000000ull
-#define SDRAM_BASE				0x7ff0000000ull
-#define SPL_TEXT_BASE				0x7ff0080000ull
+#define SDRAM_BASE				0x7FF0000000ull
+#define SPL_TEXT_BASE				0x7FF0080000ull
+#endif
+
 #define PREBOOT_VER_BASE			(SPL_TEXT_BASE - 0x1000)
 
 #define HL_UART_BAUDRATE	115200
